src/lib/skilling/skills/woodcutting.ts

```diff
@@ -1,12 +1,24 @@
+import { LootTable } from 'oldschooljs';
+
 import { Emoji } from '../../constants';
 import itemID from '../../util/itemID';
 import { Log, SkillsEnum } from '../types';
 
+const sulliuscepTable = new LootTable()
+	.add('Numulite', [4, 8], 34)
```

Comment:
```suggestion
	.add('Numulite', [1, 60], 34)
```
The loottable doesn't seem to match the wiki loot table. 
![Screenshot_20221005-043745_Chrome](https://user-images.githubusercontent.com/68188574/193969265-d557e0ed-7302-4b00-a8cd-5faaceacca1e.jpg)

Reply:
I'm not where the wiki got those numbers, but it's 100% NOT a roll from 1-60.

If I put that into the LootTable, it would be like 5x too much. Maybe there is a 1 in 1000 chance it gives 60, but I personally spent several hours doing sulliuceps myself to get a good idea of how many numulite you actually get, as well as identifying whether or not you numulite/fossils are tertiary or standard drops (standard).